[CI] SearchServiceTests class failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #3612 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/esnn3wemgvcgy)
- [elasticsearch-pull-request #27535 / part-1](https://gradle-enterprise.elastic.co/s/vt3ldgoj53lu6)
- [elasticsearch-periodic-platform-support #3608 / sles-15_platform-support-unix](https://gradle-enterprise.elastic.co/s/k3ey7v6savj6e)

**Reproduction Line:**
```
./gradlew ":server:test" --tests "org.elasticsearch.search.SearchServiceTests.testCreateSearchContextFailure" -Dtests.seed=10AEA5EEB58BA4A3 -Dtests.locale=es -Dtests.timezone=Asia/Chita -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.search.SearchServiceTests'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.search.SearchServiceTests'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: There are still [408] in-flight contexts. The first one's creation site is listed as the cause of this exception.
```

**Issue Reasons:**
- [main] 2 consecutive failures in class org.elasticsearch.search.SearchServiceTests
- [main] 3 failures in class org.elasticsearch.search.SearchServiceTests (1.9% fail rate in 158 executions)
- [main] 2 failures in pipeline elasticsearch-periodic-platform-support (50.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

I was taking a look at this earlier, but it seems to be related. The error is now: 
```
   java.lang.AssertionError: There are still [408] in-flight contexts. The first one's creation site is listed as the cause of this exception.
        at org.elasticsearch.search.MockSearchService.assertNoInFlightContext(MockSearchService.java:60)
        at org.elasticsearch.test.ESTestCase.lambda$ensureAllSearchContextsReleased$12(ESTestCase.java:817)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:1402)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:1375)
....
java.lang.AssertionError: 
    Expected: <0>
         but: was <408>
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
        at org.elasticsearch.test.ESTestCase.assertThat(ESTestCase.java:2462)
        at org.elasticsearch.test.ESTestCase.lambda$ensureAllContextsReleased$25(ESTestCase.java:2551)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:1402)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:1375)
        at org.elasticsearch.test.ESTestCase.ensureAllContextsReleased(ESTestCase.java:2550)
        at org.elasticsearch.test.ESSingleNodeTestCase.tearDown(ESSingleNodeTestCase.java:139)
```

--- a/server/src/test/java/org/elasticsearch/search/SearchServiceTests.java
+++ b/server/src/test/java/org/elasticsearch/search/SearchServiceTests.java
@@ -1548,7 +1548,7 @@ public class SearchServiceTests extends ESSingleNodeTestCase {
 
         ClearScrollRequest clearScrollRequest = new ClearScrollRequest();
         clearScrollRequest.setScrollIds(clearScrollIds);
-        client().clearScroll(clearScrollRequest);
+        client().clearScroll(clearScrollRequest).get();
 
         for (int i = 0; i < clearScrollIds.size(); i++) {
             client().prepareSearch("index").setSize(1).setScroll(TimeValue.timeValueMinutes(1)).get().decRef();

